### PR TITLE
Activate Key Changes

### PR DESCRIPTION
**Pull request by Sakshi-gupta-GL**

### CODE REVIEW

File: src/Pages/Key/Activate/ActivateKey.jsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+import { useNavigate } from "react-router-dom";
+
+import {Grid, Stack, Button} from "@mui/material";
+
+import ActivationSuccess from "./ActivationSuccess";
+import DisplayKeys from "./DisplayKeys";
+import FileUploadDetail from "./FileUploadDetail";
+import KeyActivationOptions from "./KeyActivationOptions";
+
+const ActivateKey = (props) => {
```
Comment (pinak-globallogic) [inline src/Pages/Key/Activate/ActivateKey.jsx:11]: Refactor it. Create a folder named 'Activate' inside License folder

Comment (pinak-globallogic) [inline src/Pages/Key/Activate/ActivateKey.jsx:11]: rename to ActivateLicenseWizard

File: src/Routes/index.jsx [hunk 2]
```diff
@@ -4,8 +4,10 @@ import DashboardWrapper from "Pages/Dashboard/index";
 import GenerateLicenseWrapper from "Pages/License/Generate/index";
 import SearchLicenseWrapper from "Pages/License/Search";
 import AdminPanelWrapper from "Pages/Admin";
+import KeyActivationWrapper from "Pages/Key/Activate";
 export const LICENSE_SEARCH_PAGE_ROUTE = "/license/search";
 export const ADMIN_PANEL_PAGE_ROUTE = "/license/admin";
+export const ACTIVATE_KEY_ROUTE = "/key/activate";
```
Comment (pinak-globallogic) [inline src/Routes/index.jsx:10]: /license/activate

**Review by pinak-globallogic — CHANGES_REQUESTED** [review]

File: src/Pages/Key/Activate/ActivateKey.jsx (hunk 1 above)
Comment (Sakshi-gupta-GL) [reply src/Pages/Key/Activate/ActivateKey.jsx]: done

File: src/Routes/index.jsx (hunk 2 above)
Comment (Sakshi-gupta-GL) [reply src/Routes/index.jsx]: done

**Review by pinak-globallogic — APPROVED** [review]

**Pull request merged**